### ISSUE TITLE
Night mode option

### ISSUE THREAD

**Issue by Serhioromano** [mapsme/omim#7529]
I think night mode shift is not right. When I set mode to night I have routing and map view night mode, when I set it day id both switch to day mode, but when I set it to automatic, it only set routing to night. 

I think where ever I am in the car routing or on the street walking, if I what to switch to night mode I want general map to switch. I mean do not separate to routing or map view. If you separate automatic mode switch then create 2 parameters one for routing another for map view.

Bottom line there have to be a way to automatically switch to night view on the map without routing. right now there is only manual way to switch to night mode.
